lib/ansible/plugins/connection/ssh.py

```diff
@@ -624,6 +747,33 @@ def exec_command(self, *args, **kwargs):
 
         return return_tuple
 
+    def _check_for_errors_in_return_results(self, return_code, stdout, stderr):
```

Comment:
so this function seems to be most of the 'clarification' sadly it uses message scraping, which is already very fragile, but also subject to change due to language locales.